app/views/devise/registrations/new.html.haml

```diff
@@ -0,0 +1,25 @@
+%h2 Sign up
+= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
+  = render "devise/shared/error_messages", resource: resource
+  .field
+    = f.label :name
```

Comment:
I'm not complaining about this file here because assume that you needed to add an input field for `:name`

Reply:
Yes, we added this to have the extra field for the name.